core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -457,7 +457,6 @@ object TestUtils extends Logging {
     val producerProps = props.getOrElse(new Properties)
     producerProps.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList)
     producerProps.put(ProducerConfig.ACKS_CONFIG, acks.toString)
-    producerProps.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, maxBlockMs.toString)
```

Comment:
To be able to set it to a low value in EndToEndAuthorizationTest 
to allow some test to finish quickly